File: lib/app/Models/QualityRatingDTO.php
<?php
/**
 * QualityRatingDTO
 */
namespace app\Models;

/**
 * QualityRatingDTO
 * @description Информация об индексе качества.
 */
class QualityRatingDTO {

    /** @var int $rating Значение индекса качества.*/
    public $rating = 0;

    /** @var \DateTime $calculationDate Дата вычисления.  Формат даты: &#x60;ГГГГ‑ММ‑ДД&#x60;.*/
    public $calculationDate;

    /** @var \app\Models\QualityRatingComponentDTO[] $components Составляющие индекса качества.*/
    public $components = [];

}

File: ympa_fsharp_giraffe_server/src/model/QualityRatingDTO.fs
namespace ympa_fsharp_giraffe_server.Model

open System
open System.Collections.Generic
open ympa_fsharp_giraffe_server.Model.QualityRatingComponentDTO

module QualityRatingDTO =

  //#region QualityRatingDTO


  type QualityRatingDTO = {
    Rating : int64;
    CalculationDate : DateTime;
    Components : QualityRatingComponentDTO[];
  }
  //#endregion
